src/shared/parameter/config_definitions/full_system_main_command_line_args.yaml

```diff
@@ -37,3 +37,30 @@
     description: >-
         The directory to output logs to. Absolute paths are recommended as the working directory
         is inside the bazel-out directory.
+
+- int:
```

Comment:
nit
```suggestion
- uint:
```

Reply:
Tried this but it seems using `uint` params causes a strange compilation issue, so there might be an issue with DynamicParams generation. Made a ticket to investigate: https://github.com/UBC-Thunderbots/Software/issues/2132